### ISSUE TITLE
Attack Effect Not Showing

### ISSUE THREAD

**Issue by Hellbinder1** [Wynncraft/Issues#8038]
I have the Fire Attack Effect and it isn't showing when I shoot things no matter how many times I click to activate it. My particle settings are on 'All'. Any suggestions? (I've had this effect unlocked for about 4 days and it hasn't worked once)

**Comment by thecoolkogut4** [comment]
Do you see the fire particle while you on other servers? Or do can you see the fire particle effect of someone else is using it?

**Comment by Hellbinder1** [comment]
No, I see all of the peoples particle effects like wings, storm clouds and crowns etc and can also see other peoples fire based effects but it seems that it just doesnt activate for me

**Comment by Pyromanical** [comment]
What class are you?

**Comment by Hellbinder1** [comment]
Archer
